VERSION

```diff
@@ -1 +1 @@
-0.68.0
+0.68.1
```

Comment:
I notice that your message caching also has the same version number. I don't know how much this matters,